constraints/miden-vm/hash.air

```diff
@@ -110,23 +120,29 @@ ev selector_columns(main: [s[3]]):
     let f_comp = f_abp + f_mpa + f_mva + f_mua
 
     # Enforce that selector columns are binary.
+    # Constraint degree: 2
     enf is_binary([selector]) for selector in s
 
     # Enforce that unless f_out = 1 or f_out' = 1, the values in columns s[1] and s[2] are copied 
     # over to the nex row.
+    # Constraints degree: 7
     enf is_unchanged([s[1]]) when !f_out & !f_out_next
     enf is_unchanged([s[2]]) when !f_out & !f_out_next
```

Comment:
I think there is a typo in the [docs](https://0xpolygonmiden.github.io/miden-vm/design/chiplets/hasher.html#selector-columns-constraints) regarding the degree of these constraints (imho it should be 7, not 9). The `f_out` flag has degree 3, not 4, since it is computed using only periodic column and two selectors (as opposed to `f_hout` and `f_sout` flags, which use three selectors)